crypto/README.md

```diff
@@ -0,0 +1,8 @@
+
+# Cryptography examples
+
+- [signature-demo](./signature-demo.md) ed25519 cryptographic signatures
```

Comment:
Would you mind adding a little more color into this top-level README to indicate some of the parts of these examples? The `signature-demo.md` is fantastic, this README appears to be pretty barren